incuna-sass/helpers/_link.sass

```diff
@@ -1,18 +1,19 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
-        text-decoration: underline
-        &:hover
-            text-decoration: none
     @if $type = reverse
         text-decoration: none
         &:hover
             text-decoration: underline
-    @if $type = blank
-        text-decoration: none
-        &:hover
+    @else
```

Comment:
its not wrong but it sounds wrong to have the extra else. if it hits the first one it shouldnt hit the others. unless type can be more than one.